File: VTutor.Web/Model/Tutor.cs
using System;
using System.Collections.Generic;

namespace VTutor.Model
{
    public partial class Tutor
    {
        public Guid Id { get; set; }

        public string FirstName { get; set; }
		public string LastName { get; set; }
		public string PhoneNumber { get; set; }
		public string Email { get; set; }
		public string Biography { get; set; }
		public string Education { get; set; }
		public string Languages { get; set; }
		public string Hobbies { get; set; }

		public Image TutorImage { get; set; }
		public List<Image> Documents { get; set; }

        public ApplicationUser User { get; set; }

		public List<ScheduleBlock> AvailabilitySchedule { get; set; }
        public List<Appointment> Appointments { get; set; }
        public List<Subject> Subjects { get; set; }
    }
}

File: VTutor.Web/ClientApp/app/models/Tutor.ts
export class Tutor {
	id: string;
	firstName: string;
	lastName: string;
	phoneNumber: string;
	email: string;
	biography: string;
	education: string;
	languages: string;
	hobbies: string;

	/*
		TODO: images and documents...
	*/

	subjects: TutorSubject[];
}

export class TutorSubject {
	name: Subject;
	subjectGrade: SubjectGrade;
}

export class SubjectGrade {
	name: Grade;
}

export enum Grade {
	Sixth = 6,
	Seventh = 7,
	Eighth = 8,
	Ninth = 9,
	Tenth = 10,
	Eleventh = 11,
	Twelfth = 12
}


export enum Subject {
	Math = "Math",
	English = "English",
	Spanish = "Spanish",
	French = "French",
	Statistics = "Statistics",
	Calculus = "Calculus",
	Trigonometry = "Trigonometry",
	Geometry = "Geometry",
	Algebra = "Algebra",
	Algebra2 = "Algebra 2"
}
